schema.sql

```diff
@@ -15,3 +15,24 @@ CREATE TABLE animals (
 
```

Comment:
- You have done great so far :+1: Your schema, data, and queries are all implemented correctly :+1: But there is a requirement you need to meet before moving to the next milestone. :+1: :100: 

- [ ] As required by Microverse, all your projects must be documented professionally, Currently, your readme file still contains all the placeholder comments and as required they are there to guide you and should be removed as you customize your readme. I'll suggest you go through your readme and remove all these comments.This will make your readme look professional. :+1: :100: :+1: 

- - Screenshot reference:
![Selection_058](https://github.com/hamzaw9/vet-clinic-DB/assets/107632780/768b7785-4f59-433e-b5ce-e8994381b805)
![Selection_059](https://github.com/hamzaw9/vet-clinic-DB/assets/107632780/8521b885-6864-4889-8508-38c6f04a6934)